### DISCUSSION TITLE
Exceeded the GCP free tier usage limits

### DISCUSSION THREAD

**Discussion by ankitatrey** [community/community#102624] [Discover: GitHub Best Practices]
I was using GCP(Google cloud platform) with 90 days free trial which gives $300 credit for learning purposes .

Unfortunately, there was an unintentional usage of a service named "Looker", resulting in unexpected charges above credit limit. I, therefore immediately stopped the service within 24 hours.

I was not aware of the fact that "Looker" will consume the free trial credits and the charges being started to apply automatically during my free trial and generated a bill of around $4000.

I tried to find a way to contact google but GCP basic support plan don't have dedicated support mechanism, and only provides documentation, and community supports.

Kindly guide me, how can I resolve this billing issue by possibly dropping the charges as I didn't used "Looker" for professional purpose ? 

And Is there a way to contact GCP billing team without going for $29 standard support plan ?

Actually, I tried emailing to support-in@google.com for support, but didn't get any response from there yet.

**Comment by queenofcorgis** [comment]
Hi @ankitatrey ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Google, like [their community](https://www.googlecloudcommunity.com/). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
